SBOM/spdx.md

```diff
@@ -0,0 +1,216 @@
+# [RFE] Support for SPDX SBOM generation
+
+## Introduction
+
+### Purpose of this document
+
+As a follow up on the [initial implementation of SBOM](https://github.com/AlmaLinux/build-system-rfes/blob/master/SBOM/SBOM.md), this document aims to describe the way we want to add support to [alma-sbom](https://github.com/AlmaLinux/alma-sbom) for SPDX SBOM generation.
+
+### Why SPDX?
+
+Initially, **alma-sbom** was implemented to generate SBOM documents in [OWASP CycloneDX](https://cyclonedx.org/) SBOM format, and as described in the [SBOM RFE](https://github.com/AlmaLinux/build-system-rfes/blob/master/SBOM/SBOM.md), we wanted to also extend its support to generate SBOM documents in the popular ISO-standarized format backed by the Linux Foundation, [SPDX](https://spdx.dev/).
+
+The benefit, the AlmaLinux OS community would be able to generate SBOM documents in two of the most popular SBOM standards available.
+
+## SPDX support overview
+
+### Changes to be made on alma-sbom
+
+We need to extend alma-sbom in different ways:
+
+#### 1. SPDX formatter
+
+Add an SPDX formatter in the same way we already have one for [CycloneDX](https://github.com/AlmaLinux/alma-sbom/blob/main/libsbom/cyclonedx.py)
+
+#### 2. Extending alma-sbom CLI
+
+Extend the command line options of alma-sbom in a way that we can generate SBOMs according to SPDX specification and in different output formats. Currently, we can generate SBOMs using CycloneDX into both JSON and XML formats, and we can generate SPDX SBOMs into:
+
+* JSON
+* XML
+* Tag/Value
+* YAML
+
```

Comment:
There is also support for rdf in spdx-tools, but I couldn't get it to play nice with my python version. I didn't try to find out if the problem was in my environment or in spdx-tools, but when the next stable spdx-tools is published, we should look at this again.

Reply:
Good to know, we can add this into the RFE and then deal with the problem ;)